ioc/example_test.go

```diff
@@ -3,13 +3,20 @@ package ioc
 import (
 	"context"
 	"fmt"
+	"sort"
 	"strings"
 )
 
 func ExampleGetIOCs() {
 	data := `this is a bad url http[://]google[.]com/path`
 
 	iocs := GetIOCs(data, false, true)
+	sort.SliceStable(iocs, func(i, j int) bool {
```

Comment:
What exactly is this doing? Sorting the IOCs?

Reply:
It makes the result deterministic. Without it the IOCs come out in deferent order dependent on the OS you run them on - and that breaks tests.

Reply:
Oh good to know, thanks!